config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 	// START Editing Project Variables.
 	// Project options.
 	project: 'WPGulpTheme', // Project Name.
-	projectURL: 'wpgulp.dev', // Local project URL of your already running WordPress site. Could be something like local.dev or localhost:8888.
+	projectURL: 'wpgulp.local', // Local project URL of your already running WordPress site. Could be something like wpgulp.local or localhost:8888.
```

Comment:
Maybe we should use `.test`? VVV is using it as a default TLD. Also this: https://en.wikipedia.org/wiki/.local#Microsoft_recommendations

Reply:
It's a configuration, you are meant to change it as soon as you start the project.

Reply:
I know, but then changing it to `.local` also makes no sense :D

Reply:
Yeah, it does. [Local by FlyWheel](https://local.getflywheel.com/) is a fairly popular tool to build WordPress sites on local host and it uses `.local` TLD extension.

Reply:
It's a configuration, I think VVV folks will easily change it. It's a must change one as well since it's needed for BrowserSync to work.